src/book/01-team/01-introduction/03-current-team.mdx

```diff
@@ -6,86 +6,63 @@ description: The current members of the NUbots team, and what they do.
 
-- **Lachlan Court**: Undergraduate student studying Computer Science. Team Secretary. Lead of the DevTools group. Member of the Vision group.
+- **Clayton Carlon**: Undergraduate student studying a combined degree in Electrical Engineering and Computer Systems Engineering. Hardware and audio.
 
-- **Liam Craft**: Undergraduate student studying Computer Science. Member of the DevOps and Vision groups.
+- **Peta Carlyle**: Undergraduate student studying Software Engineering.
 
-- **Nicholas Dziura**: Undergraduate student studying Computer Systems Engineering. Member of the @Home and Behaviour group.
+- **Lachlan Court**: Undergraduate student studying Computer Science. Team Secretary. Localisation, documentation and NUsight.
 
-- **Abigail Hall**: Undergraduate student studying a combined degree in Mathematics and Computer Science. Recruitment Officer. Lead of the Localisation and Odometry group. Member of the Motion group.
+- **Liam Craft**: Undergraduate student studying Computer Science. Laboratory Manager. Hardware and DevOps.
 
-- **Kip Hamiltons**: Undergraduate student studying a combined degree in Mathematics and Computer Science. Lead of the DevOps group.
+- **Jason Disher**: Undergraduate student studying Computer Science. Walk optimisation and odometry.
 
-- **Moosa Hassan**: Undergraduate student studying Software Engineering. Member of the DevOps group.
+- **Abigail Hall**: Undergraduate student studying a combined degree in Mathematics and Computer Science. Recruitment Officer. Localisation.
 
-- **Yosiah de Koeyer**: Undergraduate student studying a combined degree in Computer Systems Engineering and Computer Science. Member of the DevOps group.
+- **Thomas Legge**: Undergraduate student studying Computer Science. Speech recognition.
 
-- **Sam McFarlane**: Undergraduate student studying a combined degree in Mechatronics Engineering and Electrical Engineering. Member of the Hardware, @Home and Behaviour groups.
+- **Sam McFarlane**: Undergraduate student studying a combined degree in Mechatronics Engineering and Electrical Engineering. Hardware and odometry.
 
-- **Cameron Murtagh**: Undergraduate student studying a combined degree in Mathematics and Computer Science. Research Manager. Works on NUClear related systems.
+- **Cameron Murtagh**: Undergraduate student studying Mathematics. DevOps and software.
 
-- **Alana Noonan**: Undergraduate student studying Computer Science. Head of Arts and Crafts. Member of the Vision group, focusing on NUpbr and other dataset collection techniques.
+- **Alana Noonan**: Undergraduate student studying Computer Science. Head of Arts and Crafts. Vision data generation and benchmarking.
 
-- **Thomas O'Brien**: Undergraduate studying Mechatronics Engineering working on a Final Year Project involving the walk engine. Lead of the Motion group. Member of the Behaviour group.
+- **Thomas O'Brien**: Mechatronics Engineering PhD student. Deputy Team Leader. Motions, behaviour and localisation.
```

Comment:
I think this should be Deputy Student Team Leader, or change your one to Team Leader, for consistency between the two